curriculum/js1/8.js

```diff
@@ -5,8 +5,21 @@
  * @returns {number}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a, b, counter = 1, GCD = 1) => {
```

Comment:
never name a variable all caps. Unless its a global constant, never capitalize first character if the variable name.